api/src/wfl/api/handlers.clj

```diff
@@ -65,10 +65,12 @@
   "Return the workflows managed by the workload."
   [request]
   (log/info (select-keys request [:request-method :uri :parameters]))
-  (let [uuid (get-in request [:path-params :uuid])]
+  (let [uuid (get-in request [:path-params :uuid])
+        {:keys [status]} (get-in request [:parameters :query])]
     (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
-      (->> (workloads/load-workload-for-uuid tx uuid)
-           (workloads/workflows tx)
+      (->> (let [w (workloads/load-workload-for-uuid tx uuid)]
```

Comment:
Nit: I don't like using `w` as a variable name for a workload, and choose readability over compactness.  This is because we work primarily with workloads and workflows, and get them mixed up frequently even with their full names :)

Reply:
I see your point though in this context I'd argue it's pretty clear as he's calling workload operations on w.

Reply:
For clarity, I went ahead and updated to workload.